src/MLJMultivariateStatsInterface.jl

```diff
@@ -71,27 +71,24 @@ const SubspaceLDA_DESCR = """
     projection matrix `P = W*L` that projects a feature matrix `Xtrain` onto a lower
     dimensional space of dimension `nc - 1` such that the trace of the transformed
```

Comment:
```suggestion
    dimensional space of dimension `min(rank(Sw), nc - 1)` such that the trace of the transformed
```
The above correction is the more correct version.